SPI support
The "Serial Peripheral Interface" is a low level synchronous
protocol.  Chips that support SPI can have data transfer rates
up to several tens of Mbit/sec.  Chips are addressed with a
controller and a chipselect.  Most SPI slaves don't support
dynamic device discovery; some are even write-only or read-only.

SPI is widely used by microcontrollers to talk with sensors,
eeprom and flash memory, codecs and various other controller
chips, analog to digital (and d-to-a) converters, and more.
MMC and SD cards can be accessed using SPI protocol; and for
DataFlash cards used in MMC sockets, SPI must always be used.

SPI is one of a family of similar protocols using a four wire
interface (select, clock, data in, data out) including Microwire
(half duplex), SSP, SSI, and PSP.  This driver framework should
work with most such devices and controllers.
Depends: HAS_IOMEM
---
--- /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config.orig	2022-10-31 13:57:59.705695527 -0400
+++ /var/tmp/portage/sys-kernel/gentoo-sources-5.19.17/image/usr/src/linux-5.19.17-gentoo//.config	2022-11-01 16:19:49.829610343 -0400
@@ -1921,21 +1921,11 @@
 CONFIG_CAN_M_CAN=m
 CONFIG_CAN_M_CAN_PCI=m
 # CONFIG_CAN_M_CAN_PLATFORM is not set
-# CONFIG_CAN_M_CAN_TCAN4X5X is not set
 CONFIG_CAN_PEAK_PCIEFD=m
 # CONFIG_CAN_SJA1000 is not set
 # CONFIG_CAN_SOFTING is not set
 
 #
-# CAN SPI interfaces
-#
-CONFIG_CAN_HI311X=m
-CONFIG_CAN_MCP251X=m
-CONFIG_CAN_MCP251XFD=m
-# CONFIG_CAN_MCP251XFD_SANITY is not set
-# end of CAN SPI interfaces
-
-#
 # CAN USB interfaces
 #
 CONFIG_CAN_8DEV_USB=m
@@ -2069,7 +2059,6 @@
 CONFIG_NFC=m
 CONFIG_NFC_DIGITAL=m
 CONFIG_NFC_NCI=m
-CONFIG_NFC_NCI_SPI=m
 # CONFIG_NFC_NCI_UART is not set
 CONFIG_NFC_HCI=m
 CONFIG_NFC_SHDLC=y
@@ -2077,7 +2066,6 @@
 #
 # Near Field Communication (NFC) devices
 #
-CONFIG_NFC_TRF7970A=m
 CONFIG_NFC_MEI_PHY=m
 CONFIG_NFC_SIM=m
 CONFIG_NFC_PORT100=m
@@ -2096,16 +2084,13 @@
 CONFIG_NFC_MRVL=m
 CONFIG_NFC_MRVL_USB=m
 # CONFIG_NFC_MRVL_I2C is not set
-# CONFIG_NFC_MRVL_SPI is not set
 CONFIG_NFC_ST21NFCA=m
 CONFIG_NFC_ST21NFCA_I2C=m
 # CONFIG_NFC_ST_NCI_I2C is not set
-# CONFIG_NFC_ST_NCI_SPI is not set
 CONFIG_NFC_NXP_NCI=m
 CONFIG_NFC_NXP_NCI_I2C=m
 # CONFIG_NFC_S3FWRN5_I2C is not set
 # CONFIG_NFC_S3FWRN82_UART is not set
-# CONFIG_NFC_ST95HF is not set
 # end of Near Field Communication (NFC) devices
 
 CONFIG_PSAMPLE=m
@@ -2269,13 +2254,11 @@
 CONFIG_GENERIC_CPU_VULNERABILITIES=y
 CONFIG_REGMAP=y
 CONFIG_REGMAP_I2C=y
-CONFIG_REGMAP_SPI=m
 CONFIG_REGMAP_MMIO=y
 CONFIG_REGMAP_IRQ=y
 CONFIG_REGMAP_SOUNDWIRE=m
 CONFIG_REGMAP_SOUNDWIRE_MBQ=m
 CONFIG_REGMAP_SCCB=m
-CONFIG_REGMAP_SPI_AVMM=m
 CONFIG_DMA_SHARED_BUFFER=y
 # CONFIG_DMA_FENCE_TRACE is not set
 # end of Generic Driver Options
@@ -2416,10 +2399,6 @@
 # Self-contained MTD device drivers
 #
 # CONFIG_MTD_PMC551 is not set
-# CONFIG_MTD_DATAFLASH is not set
-# CONFIG_MTD_MCHP23K256 is not set
-CONFIG_MTD_MCHP48L640=m
-# CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
 # CONFIG_MTD_PHRAM is not set
 CONFIG_MTD_MTDRAM=m
@@ -2456,7 +2435,6 @@
 CONFIG_MTD_NAND_NANDSIM=m
 # CONFIG_MTD_NAND_RICOH is not set
 # CONFIG_MTD_NAND_DISKONCHIP is not set
-# CONFIG_MTD_SPI_NAND is not set
 
 #
 # ECC engine support
@@ -2475,7 +2453,6 @@
 # CONFIG_MTD_LPDDR is not set
 # end of LPDDR & LPDDR2 PCM memory drivers
 
-# CONFIG_MTD_SPI_NOR is not set
 CONFIG_MTD_UBI=m
 CONFIG_MTD_UBI_WL_THRESHOLD=4096
 CONFIG_MTD_UBI_BEB_LIMIT=20
@@ -2585,7 +2562,6 @@
 # CONFIG_HMC6352 is not set
 # CONFIG_DS1682 is not set
 CONFIG_VMWARE_BALLOON=m
-# CONFIG_LATTICE_ECP3_CONFIG is not set
 # CONFIG_SRAM is not set
 CONFIG_DW_XDATA_PCIE=m
 # CONFIG_PCI_ENDPOINT_TEST is not set
@@ -2597,11 +2573,9 @@
 # EEPROM support
 #
 CONFIG_EEPROM_AT24=m
-# CONFIG_EEPROM_AT25 is not set
 CONFIG_EEPROM_LEGACY=m
 CONFIG_EEPROM_MAX6875=m
 CONFIG_EEPROM_93CX6=m
-# CONFIG_EEPROM_93XX46 is not set
 CONFIG_EEPROM_IDT_89HPESX=m
 CONFIG_EEPROM_EE1004=m
 # end of EEPROM support
@@ -3030,7 +3004,6 @@
 # Distributed Switch Architecture drivers
 #
 CONFIG_B53=m
-CONFIG_B53_SPI_DRIVER=m
 CONFIG_B53_MDIO_DRIVER=m
 CONFIG_B53_MMAP_DRIVER=m
 CONFIG_B53_SRAB_DRIVER=m
@@ -3044,12 +3017,10 @@
 CONFIG_NET_DSA_MICROCHIP_KSZ_COMMON=m
 CONFIG_NET_DSA_MICROCHIP_KSZ9477=m
 # CONFIG_NET_DSA_MICROCHIP_KSZ9477_I2C is not set
-CONFIG_NET_DSA_MICROCHIP_KSZ9477_SPI=m
 # CONFIG_NET_DSA_MICROCHIP_KSZ8795 is not set
 CONFIG_NET_DSA_MV88E6XXX=m
 CONFIG_NET_DSA_MV88E6XXX_PTP=y
 # CONFIG_NET_DSA_AR9331 is not set
-# CONFIG_NET_DSA_SJA1105 is not set
 CONFIG_NET_DSA_XRS700X=m
 CONFIG_NET_DSA_XRS700X_I2C=m
 CONFIG_NET_DSA_XRS700X_MDIO=m
@@ -3062,7 +3033,6 @@
 CONFIG_NET_DSA_SMSC_LAN9303=m
 CONFIG_NET_DSA_SMSC_LAN9303_I2C=m
 CONFIG_NET_DSA_SMSC_LAN9303_MDIO=m
-# CONFIG_NET_DSA_VITESSE_VSC73XX_SPI is not set
 # CONFIG_NET_DSA_VITESSE_VSC73XX_PLATFORM is not set
 # end of Distributed Switch Architecture drivers
 
@@ -3095,8 +3065,6 @@
 CONFIG_AQTION=m
 CONFIG_NET_VENDOR_ARC=y
 CONFIG_NET_VENDOR_ASIX=y
-CONFIG_SPI_AX88796C=m
-CONFIG_SPI_AX88796C_COMPRESSION=y
 CONFIG_NET_VENDOR_ATHEROS=y
 CONFIG_ATL2=m
 CONFIG_ATL1=m
@@ -3143,7 +3111,6 @@
 CONFIG_ENIC=m
 # CONFIG_NET_VENDOR_CORTINA is not set
 CONFIG_NET_VENDOR_DAVICOM=y
-CONFIG_DM9051=m
 CONFIG_DNET=m
 CONFIG_NET_VENDOR_DEC=y
 CONFIG_NET_TULIP=y
@@ -3253,7 +3220,6 @@
 CONFIG_MLXFW=m
 CONFIG_NET_VENDOR_MICREL=y
 # CONFIG_KS8842 is not set
-# CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_KSZ884X_PCI=m
 # CONFIG_NET_VENDOR_MICROCHIP is not set
@@ -3367,7 +3333,6 @@
 # CONFIG_TI_CPSW_PHY_SEL is not set
 CONFIG_TLAN=m
 CONFIG_NET_VENDOR_VERTEXCOM=y
-CONFIG_MSE102X=m
 CONFIG_NET_VENDOR_VIA=y
 CONFIG_VIA_RHINE=m
 CONFIG_VIA_RHINE_MMIO=y
@@ -3378,7 +3343,6 @@
 # CONFIG_WIZNET_BUS_DIRECT is not set
 # CONFIG_WIZNET_BUS_INDIRECT is not set
 CONFIG_WIZNET_BUS_ANY=y
-CONFIG_WIZNET_W5100_SPI=m
 CONFIG_NET_VENDOR_XILINX=y
 CONFIG_XILINX_EMACLITE=m
 # CONFIG_XILINX_AXI_EMAC is not set
@@ -3445,7 +3409,6 @@
 # CONFIG_DP83TD510_PHY is not set
 CONFIG_VITESSE_PHY=m
 CONFIG_XILINX_GMII2RGMII=m
-# CONFIG_MICREL_KS8995MA is not set
 
 #
 # MCTP Device Drivers
@@ -3686,14 +3649,12 @@
 CONFIG_P54_COMMON=m
 CONFIG_P54_USB=m
 CONFIG_P54_PCI=m
-# CONFIG_P54_SPI is not set
 CONFIG_P54_LEDS=y
 CONFIG_WLAN_VENDOR_MARVELL=y
 CONFIG_LIBERTAS=m
 CONFIG_LIBERTAS_USB=m
 CONFIG_LIBERTAS_CS=m
 CONFIG_LIBERTAS_SDIO=m
-# CONFIG_LIBERTAS_SPI is not set
 # CONFIG_LIBERTAS_DEBUG is not set
 CONFIG_LIBERTAS_MESH=y
 # CONFIG_LIBERTAS_THINFIRM is not set
@@ -3730,7 +3691,6 @@
 CONFIG_MT7921U=m
 CONFIG_WLAN_VENDOR_MICROCHIP=y
 # CONFIG_WILC1000_SDIO is not set
-# CONFIG_WILC1000_SPI is not set
 # CONFIG_WLAN_VENDOR_PURELIFI is not set
 CONFIG_WLAN_VENDOR_RALINK=y
 CONFIG_RT2X00=m
@@ -3817,10 +3777,8 @@
 CONFIG_WLAN_VENDOR_ST=y
 CONFIG_CW1200=m
 CONFIG_CW1200_WLAN_SDIO=m
-CONFIG_CW1200_WLAN_SPI=m
 CONFIG_WLAN_VENDOR_TI=y
 CONFIG_WL1251=m
-CONFIG_WL1251_SPI=m
 CONFIG_WL1251_SDIO=m
 CONFIG_WL12XX=m
 CONFIG_WL18XX=m
@@ -3842,14 +3800,7 @@
 # CONFIG_WAN is not set
 CONFIG_IEEE802154_DRIVERS=m
 CONFIG_IEEE802154_FAKELB=m
-CONFIG_IEEE802154_AT86RF230=m
-CONFIG_IEEE802154_MRF24J40=m
-CONFIG_IEEE802154_CC2520=m
 CONFIG_IEEE802154_ATUSB=m
-CONFIG_IEEE802154_ADF7242=m
-CONFIG_IEEE802154_CA8210=m
-# CONFIG_IEEE802154_CA8210_DEBUGFS is not set
-CONFIG_IEEE802154_MCR20A=m
 # CONFIG_IEEE802154_HWSIM is not set
 
 #
@@ -3902,7 +3853,6 @@
 # CONFIG_KEYBOARD_ADC is not set
 # CONFIG_KEYBOARD_ADP5588 is not set
 # CONFIG_KEYBOARD_ADP5589 is not set
-CONFIG_KEYBOARD_APPLESPI=m
 CONFIG_KEYBOARD_ATKBD=y
 CONFIG_KEYBOARD_QT1050=m
 CONFIG_KEYBOARD_QT1070=m
@@ -3988,8 +3938,6 @@
 CONFIG_JOYSTICK_XPAD_FF=y
 CONFIG_JOYSTICK_XPAD_LEDS=y
 CONFIG_JOYSTICK_WALKERA0701=m
-CONFIG_JOYSTICK_PSXPAD_SPI=m
-CONFIG_JOYSTICK_PSXPAD_SPI_FF=y
 CONFIG_JOYSTICK_PXRC=m
 CONFIG_JOYSTICK_QWIIC=m
 # CONFIG_JOYSTICK_FSIA6B is not set
@@ -4002,8 +3950,6 @@
 CONFIG_TABLET_USB_PEGASUS=m
 CONFIG_TABLET_SERIAL_WACOM4=m
 CONFIG_INPUT_TOUCHSCREEN=y
-# CONFIG_TOUCHSCREEN_ADS7846 is not set
-# CONFIG_TOUCHSCREEN_AD7877 is not set
 # CONFIG_TOUCHSCREEN_AD7879 is not set
 # CONFIG_TOUCHSCREEN_ADC is not set
 CONFIG_TOUCHSCREEN_ATMEL_MXT=m
@@ -4072,7 +4018,6 @@
 CONFIG_TOUCHSCREEN_TOUCHIT213=m
 CONFIG_TOUCHSCREEN_TSC_SERIO=m
 # CONFIG_TOUCHSCREEN_TSC2004 is not set
-# CONFIG_TOUCHSCREEN_TSC2005 is not set
 CONFIG_TOUCHSCREEN_TSC2007=m
 CONFIG_TOUCHSCREEN_TSC2007_IIO=y
 CONFIG_TOUCHSCREEN_RM_TS=m
@@ -4081,7 +4026,6 @@
 CONFIG_TOUCHSCREEN_ST1232=m
 # CONFIG_TOUCHSCREEN_STMFTS is not set
 # CONFIG_TOUCHSCREEN_SUR40 is not set
-CONFIG_TOUCHSCREEN_SURFACE3_SPI=m
 # CONFIG_TOUCHSCREEN_SX8654 is not set
 # CONFIG_TOUCHSCREEN_TPS6507X is not set
 CONFIG_TOUCHSCREEN_ZET6223=m
@@ -4091,7 +4035,6 @@
 CONFIG_TOUCHSCREEN_ZINITIX=m
 CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_AD714X is not set
-# CONFIG_INPUT_ARIZONA_HAPTICS is not set
 # CONFIG_INPUT_BMA150 is not set
 CONFIG_INPUT_E3X0_BUTTON=m
 CONFIG_INPUT_PCSPKR=m
@@ -4130,7 +4073,6 @@
 # CONFIG_INPUT_DRV2667_HAPTICS is not set
 CONFIG_RMI4_CORE=m
 CONFIG_RMI4_I2C=m
-CONFIG_RMI4_SPI=m
 CONFIG_RMI4_SMB=m
 CONFIG_RMI4_F03=y
 CONFIG_RMI4_F03_SERIO=m
@@ -4214,18 +4156,14 @@
 #
 # Non-8250 serial port support
 #
-# CONFIG_SERIAL_MAX3100 is not set
-# CONFIG_SERIAL_MAX310X is not set
 # CONFIG_SERIAL_UARTLITE is not set
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 CONFIG_SERIAL_JSM=m
 # CONFIG_SERIAL_LANTIQ is not set
 # CONFIG_SERIAL_SCCNXP is not set
-CONFIG_SERIAL_SC16IS7XX_CORE=m
 CONFIG_SERIAL_SC16IS7XX=m
 # CONFIG_SERIAL_SC16IS7XX_I2C is not set
-CONFIG_SERIAL_SC16IS7XX_SPI=y
 # CONFIG_SERIAL_ALTERA_JTAGUART is not set
 # CONFIG_SERIAL_ALTERA_UART is not set
 CONFIG_SERIAL_ARC=m
@@ -4298,8 +4236,6 @@
 CONFIG_HW_RANDOM_TPM=y
 CONFIG_TCG_TIS_CORE=y
 CONFIG_TCG_TIS=y
-CONFIG_TCG_TIS_SPI=m
-CONFIG_TCG_TIS_SPI_CR50=y
 CONFIG_TCG_TIS_I2C_CR50=m
 CONFIG_TCG_TIS_I2C_ATMEL=m
 CONFIG_TCG_TIS_I2C_INFINEON=m
@@ -4311,7 +4247,6 @@
 CONFIG_TCG_CRB=y
 CONFIG_TCG_VTPM_PROXY=m
 # CONFIG_TCG_TIS_ST33ZP24_I2C is not set
-# CONFIG_TCG_TIS_ST33ZP24_SPI is not set
 CONFIG_TELCLOCK=m
 CONFIG_XILLYBUS_CLASS=m
 CONFIG_XILLYBUS=m
@@ -4427,51 +4362,7 @@
 # end of I2C support
 
 # CONFIG_I3C is not set
-CONFIG_SPI=y
-CONFIG_SPI_MASTER=y
-# CONFIG_SPI_MEM is not set
-
-#
-# SPI Master Controller Drivers
-#
-# CONFIG_SPI_ALTERA is not set
-CONFIG_SPI_ALTERA_CORE=m
-CONFIG_SPI_ALTERA_DFL=m
-# CONFIG_SPI_AXI_SPI_ENGINE is not set
-# CONFIG_SPI_BITBANG is not set
-# CONFIG_SPI_BUTTERFLY is not set
-# CONFIG_SPI_CADENCE is not set
-# CONFIG_SPI_DESIGNWARE is not set
-CONFIG_SPI_DLN2=m
-# CONFIG_SPI_NXP_FLEXSPI is not set
-# CONFIG_SPI_GPIO is not set
-# CONFIG_SPI_LM70_LLP is not set
-# CONFIG_SPI_LANTIQ_SSC is not set
-# CONFIG_SPI_OC_TINY is not set
-CONFIG_SPI_PXA2XX=m
-CONFIG_SPI_PXA2XX_PCI=m
-# CONFIG_SPI_ROCKCHIP is not set
-# CONFIG_SPI_SC18IS602 is not set
-# CONFIG_SPI_SIFIVE is not set
-# CONFIG_SPI_MXIC is not set
-# CONFIG_SPI_XCOMM is not set
-# CONFIG_SPI_XILINX is not set
-# CONFIG_SPI_ZYNQMP_GQSPI is not set
-CONFIG_SPI_AMD=y
-
-#
-# SPI Multiplexer support
-#
-CONFIG_SPI_MUX=m
-
-#
-# SPI Protocol Masters
-#
-CONFIG_SPI_SPIDEV=m
-# CONFIG_SPI_LOOPBACK_TEST is not set
-# CONFIG_SPI_TLE62X0 is not set
-# CONFIG_SPI_SLAVE is not set
-CONFIG_SPI_DYNAMIC=y
+# CONFIG_SPI is not set
 # CONFIG_SPMI is not set
 # CONFIG_HSI is not set
 CONFIG_PPS=y
@@ -4587,7 +4478,6 @@
 #
 # MFD GPIO expanders
 #
-CONFIG_GPIO_ARIZONA=m
 CONFIG_GPIO_BD9571MWV=m
 CONFIG_GPIO_CRYSTAL_COVE=y
 CONFIG_GPIO_DLN2=m
@@ -4606,16 +4496,6 @@
 # end of PCI GPIO expanders
 
 #
-# SPI GPIO expanders
-#
-# CONFIG_GPIO_MAX3191X is not set
-# CONFIG_GPIO_MAX7301 is not set
-# CONFIG_GPIO_MC33880 is not set
-# CONFIG_GPIO_PISOSR is not set
-# CONFIG_GPIO_XRA1403 is not set
-# end of SPI GPIO expanders
-
-#
 # USB GPIO expanders
 #
 # end of USB GPIO expanders
@@ -4732,7 +4612,6 @@
 #
 CONFIG_SENSORS_ABITUGURU=m
 CONFIG_SENSORS_ABITUGURU3=m
-CONFIG_SENSORS_AD7314=m
 CONFIG_SENSORS_AD7414=m
 CONFIG_SENSORS_AD7418=m
 CONFIG_SENSORS_ADM1021=m
@@ -4743,7 +4622,6 @@
 # CONFIG_SENSORS_ADM1177 is not set
 CONFIG_SENSORS_ADM9240=m
 CONFIG_SENSORS_ADT7X10=m
-CONFIG_SENSORS_ADT7310=m
 CONFIG_SENSORS_ADT7410=m
 CONFIG_SENSORS_ADT7411=m
 CONFIG_SENSORS_ADT7462=m
@@ -4791,7 +4669,6 @@
 CONFIG_SENSORS_LTC2945=m
 CONFIG_SENSORS_LTC2947=m
 CONFIG_SENSORS_LTC2947_I2C=m
-CONFIG_SENSORS_LTC2947_SPI=m
 CONFIG_SENSORS_LTC2990=m
 # CONFIG_SENSORS_LTC2992 is not set
 CONFIG_SENSORS_LTC4151=m
@@ -4800,13 +4677,11 @@
 CONFIG_SENSORS_LTC4245=m
 CONFIG_SENSORS_LTC4260=m
 CONFIG_SENSORS_LTC4261=m
-CONFIG_SENSORS_MAX1111=m
 # CONFIG_SENSORS_MAX127 is not set
 CONFIG_SENSORS_MAX16065=m
 CONFIG_SENSORS_MAX1619=m
 CONFIG_SENSORS_MAX1668=m
 CONFIG_SENSORS_MAX197=m
-CONFIG_SENSORS_MAX31722=m
 # CONFIG_SENSORS_MAX31730 is not set
 CONFIG_SENSORS_MAX6620=m
 # CONFIG_SENSORS_MAX6621 is not set
@@ -4820,9 +4695,7 @@
 CONFIG_SENSORS_TC654=m
 # CONFIG_SENSORS_TPS23861 is not set
 CONFIG_SENSORS_MR75203=m
-CONFIG_SENSORS_ADCXX=m
 CONFIG_SENSORS_LM63=m
-CONFIG_SENSORS_LM70=m
 CONFIG_SENSORS_LM73=m
 CONFIG_SENSORS_LM75=m
 CONFIG_SENSORS_LM77=m
@@ -4919,7 +4792,6 @@
 # CONFIG_SENSORS_SMM665 is not set
 CONFIG_SENSORS_ADC128D818=m
 CONFIG_SENSORS_ADS7828=m
-CONFIG_SENSORS_ADS7871=m
 CONFIG_SENSORS_AMC6821=m
 CONFIG_SENSORS_INA209=m
 CONFIG_SENSORS_INA2XX=m
@@ -4950,7 +4822,6 @@
 CONFIG_SENSORS_W83627HF=m
 CONFIG_SENSORS_W83627EHF=m
 # CONFIG_SENSORS_XGENE is not set
-CONFIG_SENSORS_INTEL_M10_BMC_HWMON=m
 
 #
 # ACPI drivers
@@ -5114,14 +4985,12 @@
 CONFIG_MFD_CROS_EC_DEV=m
 # CONFIG_MFD_MADERA is not set
 # CONFIG_PMIC_DA903X is not set
-# CONFIG_MFD_DA9052_SPI is not set
 # CONFIG_MFD_DA9052_I2C is not set
 # CONFIG_MFD_DA9055 is not set
 # CONFIG_MFD_DA9062 is not set
 # CONFIG_MFD_DA9063 is not set
 # CONFIG_MFD_DA9150 is not set
 CONFIG_MFD_DLN2=m
-# CONFIG_MFD_MC13XXX_SPI is not set
 # CONFIG_MFD_MC13XXX_I2C is not set
 # CONFIG_MFD_MP2629 is not set
 # CONFIG_HTC_PASIC3 is not set
@@ -5154,7 +5023,6 @@
 # CONFIG_MFD_MT6360 is not set
 # CONFIG_MFD_MT6397 is not set
 # CONFIG_MFD_MENF21BMC is not set
-# CONFIG_EZX_PCAP is not set
 # CONFIG_MFD_VIPERBOARD is not set
 # CONFIG_MFD_RETU is not set
 # CONFIG_MFD_PCF50633 is not set
@@ -5183,29 +5051,19 @@
 # CONFIG_MFD_TPS6586X is not set
 # CONFIG_MFD_TPS65910 is not set
 # CONFIG_MFD_TPS65912_I2C is not set
-# CONFIG_MFD_TPS65912_SPI is not set
 # CONFIG_TWL4030_CORE is not set
 # CONFIG_TWL6040_CORE is not set
 CONFIG_MFD_WL1273_CORE=m
 # CONFIG_MFD_LM3533 is not set
 # CONFIG_MFD_TQMX86 is not set
 CONFIG_MFD_VX855=m
-CONFIG_MFD_ARIZONA=m
 # CONFIG_MFD_ARIZONA_I2C is not set
-CONFIG_MFD_ARIZONA_SPI=m
-# CONFIG_MFD_CS47L24 is not set
-CONFIG_MFD_WM5102=y
-# CONFIG_MFD_WM5110 is not set
-# CONFIG_MFD_WM8997 is not set
-# CONFIG_MFD_WM8998 is not set
 # CONFIG_MFD_WM8400 is not set
 # CONFIG_MFD_WM831X_I2C is not set
-# CONFIG_MFD_WM831X_SPI is not set
 # CONFIG_MFD_WM8350_I2C is not set
 # CONFIG_MFD_WM8994 is not set
 # CONFIG_MFD_ATC260X_I2C is not set
 # CONFIG_RAVE_SP_CORE is not set
-CONFIG_MFD_INTEL_M10_BMC=m
 # end of Multifunction device drivers
 
 CONFIG_REGULATOR=y
@@ -5216,8 +5074,6 @@
 # CONFIG_REGULATOR_88PG86X is not set
 # CONFIG_REGULATOR_ACT8865 is not set
 # CONFIG_REGULATOR_AD5398 is not set
-CONFIG_REGULATOR_ARIZONA_LDO1=m
-CONFIG_REGULATOR_ARIZONA_MICSUPP=m
 # CONFIG_REGULATOR_AXP20X is not set
 # CONFIG_REGULATOR_BD9571MWV is not set
 # CONFIG_REGULATOR_DA9210 is not set
@@ -5262,7 +5118,6 @@
 # CONFIG_REGULATOR_TPS65023 is not set
 # CONFIG_REGULATOR_TPS6507X is not set
 # CONFIG_REGULATOR_TPS65132 is not set
-# CONFIG_REGULATOR_TPS6524X is not set
 CONFIG_REGULATOR_TPS68470=m
 CONFIG_RC_CORE=y
 CONFIG_BPF_LIRC_MODE2=y
@@ -5596,7 +5451,6 @@
 CONFIG_MANTIS_CORE=m
 CONFIG_DVB_MANTIS=m
 CONFIG_DVB_HOPPER=m
-CONFIG_DVB_NETUP_UNIDVB=m
 CONFIG_DVB_NGENE=m
 CONFIG_DVB_PLUTO2=m
 CONFIG_DVB_PT1=m
@@ -5824,7 +5678,6 @@
 CONFIG_VIDEO_RDACM20=m
 # CONFIG_VIDEO_RDACM21 is not set
 CONFIG_VIDEO_RJ54N1=m
-CONFIG_VIDEO_S5C73M3=m
 CONFIG_VIDEO_S5K4ECGX=m
 CONFIG_VIDEO_S5K5BAF=m
 CONFIG_VIDEO_S5K6A3=m
@@ -5888,18 +5741,6 @@
 CONFIG_VIDEO_UPD64083=m
 CONFIG_VIDEO_SAA6752HS=m
 CONFIG_VIDEO_M52790=m
-
-#
-# SPI I2C drivers auto-selected by 'Autoselect ancillary drivers'
-#
-
-#
-# Media SPI Adapters
-#
-CONFIG_CXD2880_SPI_DRV=m
-CONFIG_VIDEO_GS1662=m
-# end of Media SPI Adapters
-
 CONFIG_MEDIA_TUNER=m
 
 #
@@ -6076,10 +5917,7 @@
 #
 CONFIG_DVB_A8293=m
 CONFIG_DVB_AF9033=m
-CONFIG_DVB_ASCOT2E=m
 CONFIG_DVB_ATBM8830=m
-CONFIG_DVB_HELENE=m
-CONFIG_DVB_HORUS3A=m
 CONFIG_DVB_ISL6405=m
 CONFIG_DVB_ISL6421=m
 CONFIG_DVB_ISL6423=m
@@ -6110,7 +5948,6 @@
 CONFIG_INTEL_GTT=y
 CONFIG_VGA_SWITCHEROO=y
 CONFIG_DRM=y
-CONFIG_DRM_MIPI_DBI=m
 CONFIG_DRM_MIPI_DSI=y
 # CONFIG_DRM_DEBUG_MM is not set
 CONFIG_DRM_KMS_HELPER=m
@@ -6127,7 +5964,6 @@
 CONFIG_DRM_BUDDY=m
 CONFIG_DRM_VRAM_HELPER=m
 CONFIG_DRM_TTM_HELPER=m
-CONFIG_DRM_GEM_CMA_HELPER=m
 CONFIG_DRM_GEM_SHMEM_HELPER=m
 CONFIG_DRM_SCHED=m
 
@@ -6211,7 +6047,6 @@
 # Display Panels
 #
 # CONFIG_DRM_PANEL_RASPBERRYPI_TOUCHSCREEN is not set
-CONFIG_DRM_PANEL_WIDECHIPS_WS2401=m
 # end of Display Panels
 
 CONFIG_DRM_BRIDGE=y
@@ -6228,23 +6063,12 @@
 CONFIG_DRM_BOCHS=m
 CONFIG_DRM_CIRRUS_QEMU=m
 CONFIG_DRM_GM12U320=m
-CONFIG_DRM_PANEL_MIPI_DBI=m
 # CONFIG_DRM_SIMPLEDRM is not set
-# CONFIG_TINYDRM_HX8357D is not set
-CONFIG_TINYDRM_ILI9163=m
-# CONFIG_TINYDRM_ILI9225 is not set
-# CONFIG_TINYDRM_ILI9341 is not set
-CONFIG_TINYDRM_ILI9486=m
-# CONFIG_TINYDRM_MI0283QT is not set
-# CONFIG_TINYDRM_REPAPER is not set
-# CONFIG_TINYDRM_ST7586 is not set
-# CONFIG_TINYDRM_ST7735R is not set
 # CONFIG_DRM_XEN_FRONTEND is not set
 CONFIG_DRM_VBOXVIDEO=m
 CONFIG_DRM_GUD=m
 CONFIG_DRM_SSD130X=m
 CONFIG_DRM_SSD130X_I2C=m
-CONFIG_DRM_SSD130X_SPI=m
 CONFIG_DRM_HYPERV=m
 # CONFIG_DRM_LEGACY is not set
 CONFIG_DRM_PANEL_ORIENTATION_QUIRKS=y
@@ -6327,18 +6151,7 @@
 # Backlight & LCD device support
 #
 CONFIG_LCD_CLASS_DEVICE=m
-# CONFIG_LCD_L4F00242T03 is not set
-# CONFIG_LCD_LMS283GF05 is not set
-# CONFIG_LCD_LTV350QV is not set
-# CONFIG_LCD_ILI922X is not set
-# CONFIG_LCD_ILI9320 is not set
-# CONFIG_LCD_TDO24M is not set
-# CONFIG_LCD_VGG2432A4 is not set
 CONFIG_LCD_PLATFORM=m
-# CONFIG_LCD_AMS369FG06 is not set
-# CONFIG_LCD_LMS501KF03 is not set
-# CONFIG_LCD_HX8357 is not set
-# CONFIG_LCD_OTM3225A is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
 CONFIG_BACKLIGHT_KTD253=m
 CONFIG_BACKLIGHT_PWM=m
@@ -6357,7 +6170,6 @@
 CONFIG_BACKLIGHT_ARCXCNN=m
 # end of Backlight & LCD device support
 
-CONFIG_VIDEOMODE_HELPERS=y
 CONFIG_HDMI=y
 
 #
@@ -6523,7 +6335,6 @@
 CONFIG_SND_HDA_PATCH_LOADER=y
 CONFIG_SND_HDA_SCODEC_CS35L41=m
 CONFIG_SND_HDA_SCODEC_CS35L41_I2C=m
-CONFIG_SND_HDA_SCODEC_CS35L41_SPI=m
 CONFIG_SND_HDA_CODEC_REALTEK=m
 CONFIG_SND_HDA_CODEC_ANALOG=m
 CONFIG_SND_HDA_CODEC_SIGMATEL=m
@@ -6552,7 +6363,6 @@
 CONFIG_SND_INTEL_DSP_CONFIG=m
 CONFIG_SND_INTEL_SOUNDWIRE_ACPI=m
 # CONFIG_SND_INTEL_BYT_PREFER_SOF is not set
-# CONFIG_SND_SPI is not set
 CONFIG_SND_USB=y
 CONFIG_SND_USB_AUDIO=m
 CONFIG_SND_USB_AUDIO_USE_MEDIA_CONTROLLER=y
@@ -6600,7 +6410,6 @@
 CONFIG_SND_SOC_AMD_RENOIR=m
 CONFIG_SND_SOC_AMD_RENOIR_MACH=m
 CONFIG_SND_SOC_AMD_ACP5x=m
-CONFIG_SND_SOC_AMD_VANGOGH_MACH=m
 CONFIG_SND_SOC_AMD_ACP6x=m
 CONFIG_SND_SOC_AMD_YC_MACH=m
 CONFIG_SND_AMD_ACP_CONFIG=m
@@ -6658,11 +6467,9 @@
 CONFIG_SND_SOC_INTEL_SOF_CIRRUS_COMMON=m
 CONFIG_SND_SOC_INTEL_HASWELL_MACH=m
 CONFIG_SND_SOC_INTEL_BDW_RT5650_MACH=m
-CONFIG_SND_SOC_INTEL_BDW_RT5677_MACH=m
 CONFIG_SND_SOC_INTEL_BROADWELL_MACH=m
 CONFIG_SND_SOC_INTEL_BYTCR_RT5640_MACH=m
 CONFIG_SND_SOC_INTEL_BYTCR_RT5651_MACH=m
-CONFIG_SND_SOC_INTEL_BYTCR_WM5102_MACH=m
 CONFIG_SND_SOC_INTEL_CHT_BSW_RT5672_MACH=m
 CONFIG_SND_SOC_INTEL_CHT_BSW_RT5645_MACH=m
 CONFIG_SND_SOC_INTEL_CHT_BSW_MAX98090_TI_MACH=m
@@ -6680,7 +6487,6 @@
 CONFIG_SND_SOC_INTEL_BXT_RT298_MACH=m
 CONFIG_SND_SOC_INTEL_SOF_WM8804_MACH=m
 CONFIG_SND_SOC_INTEL_KBL_RT5663_MAX98927_MACH=m
-CONFIG_SND_SOC_INTEL_KBL_RT5663_RT5514_MAX98927_MACH=m
 CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98357A_MACH=m
 CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98927_MACH=m
 CONFIG_SND_SOC_INTEL_KBL_RT5660_MACH=m
@@ -6756,22 +6562,18 @@
 #
 # CODEC drivers
 #
-CONFIG_SND_SOC_ARIZONA=m
 CONFIG_SND_SOC_WM_ADSP=m
 CONFIG_SND_SOC_AC97_CODEC=m
 CONFIG_SND_SOC_ADAU_UTILS=m
 # CONFIG_SND_SOC_ADAU1372_I2C is not set
-# CONFIG_SND_SOC_ADAU1372_SPI is not set
 # CONFIG_SND_SOC_ADAU1701 is not set
 CONFIG_SND_SOC_ADAU17X1=m
 CONFIG_SND_SOC_ADAU1761=m
 CONFIG_SND_SOC_ADAU1761_I2C=m
-CONFIG_SND_SOC_ADAU1761_SPI=m
 CONFIG_SND_SOC_ADAU7002=m
 CONFIG_SND_SOC_ADAU7118=m
 CONFIG_SND_SOC_ADAU7118_HW=m
 CONFIG_SND_SOC_ADAU7118_I2C=m
-# CONFIG_SND_SOC_AK4104 is not set
 # CONFIG_SND_SOC_AK4118 is not set
 # CONFIG_SND_SOC_AK4375 is not set
 # CONFIG_SND_SOC_AK4458 is not set
@@ -6792,11 +6594,9 @@
 CONFIG_SND_SOC_CS35L36=m
 CONFIG_SND_SOC_CS35L41_LIB=m
 CONFIG_SND_SOC_CS35L41=m
-CONFIG_SND_SOC_CS35L41_SPI=m
 CONFIG_SND_SOC_CS35L41_I2C=m
 CONFIG_SND_SOC_CS35L45_TABLES=m
 CONFIG_SND_SOC_CS35L45=m
-CONFIG_SND_SOC_CS35L45_SPI=m
 CONFIG_SND_SOC_CS35L45_I2C=m
 CONFIG_SND_SOC_CS42L42=m
 # CONFIG_SND_SOC_CS42L51_I2C is not set
@@ -6807,7 +6607,6 @@
 # CONFIG_SND_SOC_CS4265 is not set
 # CONFIG_SND_SOC_CS4270 is not set
 # CONFIG_SND_SOC_CS4271_I2C is not set
-# CONFIG_SND_SOC_CS4271_SPI is not set
 # CONFIG_SND_SOC_CS42XX8_I2C is not set
 CONFIG_SND_SOC_CS43130=m
 # CONFIG_SND_SOC_CS4341 is not set
@@ -6822,7 +6621,6 @@
 CONFIG_SND_SOC_ES8316=m
 CONFIG_SND_SOC_ES8328=m
 CONFIG_SND_SOC_ES8328_I2C=m
-CONFIG_SND_SOC_ES8328_SPI=m
 # CONFIG_SND_SOC_GTM601 is not set
 CONFIG_SND_SOC_HDAC_HDMI=m
 CONFIG_SND_SOC_HDAC_HDA=m
@@ -6846,19 +6644,14 @@
 CONFIG_SND_SOC_PCM1789=m
 CONFIG_SND_SOC_PCM1789_I2C=m
 # CONFIG_SND_SOC_PCM179X_I2C is not set
-# CONFIG_SND_SOC_PCM179X_SPI is not set
 CONFIG_SND_SOC_PCM186X=m
 CONFIG_SND_SOC_PCM186X_I2C=m
-CONFIG_SND_SOC_PCM186X_SPI=m
 CONFIG_SND_SOC_PCM3060=m
 CONFIG_SND_SOC_PCM3060_I2C=m
-CONFIG_SND_SOC_PCM3060_SPI=m
 # CONFIG_SND_SOC_PCM3168A_I2C is not set
-# CONFIG_SND_SOC_PCM3168A_SPI is not set
 # CONFIG_SND_SOC_PCM5102A is not set
 CONFIG_SND_SOC_PCM512x=m
 CONFIG_SND_SOC_PCM512x_I2C=m
-# CONFIG_SND_SOC_PCM512x_SPI is not set
 # CONFIG_SND_SOC_RK3328 is not set
 CONFIG_SND_SOC_RL6231=m
 CONFIG_SND_SOC_RL6347A=m
@@ -6870,8 +6663,6 @@
 CONFIG_SND_SOC_RT1308=m
 CONFIG_SND_SOC_RT1308_SDW=m
 CONFIG_SND_SOC_RT1316_SDW=m
-CONFIG_SND_SOC_RT5514=m
-CONFIG_SND_SOC_RT5514_SPI=m
 # CONFIG_SND_SOC_RT5616 is not set
 # CONFIG_SND_SOC_RT5631 is not set
 CONFIG_SND_SOC_RT5640=m
@@ -6881,8 +6672,6 @@
 CONFIG_SND_SOC_RT5660=m
 CONFIG_SND_SOC_RT5663=m
 CONFIG_SND_SOC_RT5670=m
-CONFIG_SND_SOC_RT5677=m
-CONFIG_SND_SOC_RT5677_SPI=m
 CONFIG_SND_SOC_RT5682=m
 CONFIG_SND_SOC_RT5682_I2C=m
 CONFIG_SND_SOC_RT5682_SDW=m
@@ -6905,7 +6694,6 @@
 CONFIG_SND_SOC_SPDIF=m
 # CONFIG_SND_SOC_SSM2305 is not set
 # CONFIG_SND_SOC_SSM2518 is not set
-# CONFIG_SND_SOC_SSM2602_SPI is not set
 # CONFIG_SND_SOC_SSM2602_I2C is not set
 CONFIG_SND_SOC_SSM4567=m
 # CONFIG_SND_SOC_STA32X is not set
@@ -6925,20 +6713,16 @@
 # CONFIG_SND_SOC_TFA989X is not set
 CONFIG_SND_SOC_TLV320ADC3XXX=m
 # CONFIG_SND_SOC_TLV320AIC23_I2C is not set
-# CONFIG_SND_SOC_TLV320AIC23_SPI is not set
 # CONFIG_SND_SOC_TLV320AIC31XX is not set
 CONFIG_SND_SOC_TLV320AIC32X4=m
 CONFIG_SND_SOC_TLV320AIC32X4_I2C=m
-CONFIG_SND_SOC_TLV320AIC32X4_SPI=m
 # CONFIG_SND_SOC_TLV320AIC3X_I2C is not set
-# CONFIG_SND_SOC_TLV320AIC3X_SPI is not set
 CONFIG_SND_SOC_TLV320ADCX140=m
 CONFIG_SND_SOC_TS3A227E=m
 CONFIG_SND_SOC_TSCS42XX=m
 # CONFIG_SND_SOC_TSCS454 is not set
 # CONFIG_SND_SOC_UDA1334 is not set
 # CONFIG_SND_SOC_WCD938X_SDW is not set
-CONFIG_SND_SOC_WM5102=m
 # CONFIG_SND_SOC_WM8510 is not set
 # CONFIG_SND_SOC_WM8523 is not set
 CONFIG_SND_SOC_WM8524=m
@@ -6947,17 +6731,14 @@
 # CONFIG_SND_SOC_WM8728 is not set
 CONFIG_SND_SOC_WM8731=m
 CONFIG_SND_SOC_WM8731_I2C=m
-CONFIG_SND_SOC_WM8731_SPI=m
 # CONFIG_SND_SOC_WM8737 is not set
 # CONFIG_SND_SOC_WM8741 is not set
 # CONFIG_SND_SOC_WM8750 is not set
 # CONFIG_SND_SOC_WM8753 is not set
-# CONFIG_SND_SOC_WM8770 is not set
 # CONFIG_SND_SOC_WM8776 is not set
 # CONFIG_SND_SOC_WM8782 is not set
 CONFIG_SND_SOC_WM8804=m
 CONFIG_SND_SOC_WM8804_I2C=m
-# CONFIG_SND_SOC_WM8804_SPI is not set
 # CONFIG_SND_SOC_WM8903 is not set
 # CONFIG_SND_SOC_WM8904 is not set
 CONFIG_SND_SOC_WM8940=m
@@ -6967,7 +6748,6 @@
 # CONFIG_SND_SOC_WM8978 is not set
 # CONFIG_SND_SOC_WM8985 is not set
 # CONFIG_SND_SOC_WSA881X is not set
-CONFIG_SND_SOC_ZL38060=m
 CONFIG_SND_SOC_MAX9759=m
 # CONFIG_SND_SOC_MT6351 is not set
 # CONFIG_SND_SOC_MT6358 is not set
@@ -7213,7 +6993,6 @@
 # CONFIG_USB_OXU210HP_HCD is not set
 # CONFIG_USB_ISP116X_HCD is not set
 # CONFIG_USB_FOTG210_HCD is not set
-# CONFIG_USB_MAX3421_HCD is not set
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_OHCI_HCD_PCI=y
 # CONFIG_USB_OHCI_HCD_PLATFORM is not set
@@ -7443,7 +7222,6 @@
 CONFIG_MMC_WBSD=m
 CONFIG_MMC_ALCOR=m
 CONFIG_MMC_TIFM_SD=m
-# CONFIG_MMC_SPI is not set
 CONFIG_MMC_SDRICOH_CS=m
 CONFIG_MMC_CB710=m
 CONFIG_MMC_VIA_SDMMC=m
@@ -7498,7 +7276,6 @@
 CONFIG_LEDS_CLEVO_MAIL=m
 # CONFIG_LEDS_PCA955X is not set
 # CONFIG_LEDS_PCA963X is not set
-# CONFIG_LEDS_DAC124S085 is not set
 # CONFIG_LEDS_PWM is not set
 # CONFIG_LEDS_REGULATOR is not set
 # CONFIG_LEDS_BD2802 is not set
@@ -7702,20 +7479,6 @@
 #
 # SPI RTC drivers
 #
-CONFIG_RTC_DRV_M41T93=m
-CONFIG_RTC_DRV_M41T94=m
-# CONFIG_RTC_DRV_DS1302 is not set
-CONFIG_RTC_DRV_DS1305=m
-CONFIG_RTC_DRV_DS1343=m
-CONFIG_RTC_DRV_DS1347=m
-CONFIG_RTC_DRV_DS1390=m
-CONFIG_RTC_DRV_MAX6916=m
-CONFIG_RTC_DRV_R9701=m
-CONFIG_RTC_DRV_RX4581=m
-CONFIG_RTC_DRV_RS5C348=m
-CONFIG_RTC_DRV_MAX6902=m
-CONFIG_RTC_DRV_PCF2123=m
-CONFIG_RTC_DRV_MCP795=m
 CONFIG_RTC_I2C_AND_SPI=y
 
 #
@@ -7973,14 +7736,11 @@
 #
 # Accelerometers
 #
-# CONFIG_ADIS16203 is not set
-# CONFIG_ADIS16240 is not set
 # end of Accelerometers
 
 #
 # Analog to digital converters
 #
-# CONFIG_AD7816 is not set
 # end of Analog to digital converters
 
 #
@@ -7998,8 +7758,6 @@
 #
 # Direct Digital Synthesis
 #
-# CONFIG_AD9832 is not set
-# CONFIG_AD9834 is not set
 # end of Direct Digital Synthesis
 
 #
@@ -8017,7 +7775,6 @@
 #
 # Resolver to digital converters
 #
-# CONFIG_AD2S1210 is not set
 # end of Resolver to digital converters
 # end of IIO staging drivers
 
@@ -8029,9 +7786,7 @@
 # CONFIG_DVB_AV7110 is not set
 # CONFIG_LTE_GDM724X is not set
 # CONFIG_FIREWIRE_SERIAL is not set
-# CONFIG_FB_TFT is not set
 # CONFIG_KS7010 is not set
-# CONFIG_PI433 is not set
 # CONFIG_FIELDBUS_DEV is not set
 CONFIG_QLGE=m
 
@@ -8158,10 +7913,8 @@
 CONFIG_SONYPI_COMPAT=y
 CONFIG_SYSTEM76_ACPI=m
 CONFIG_TOPSTAR_LAPTOP=m
-CONFIG_SERIAL_MULTI_INSTANTIATE=m
 CONFIG_MLX_PLATFORM=m
 CONFIG_TOUCHSCREEN_DMI=y
-CONFIG_X86_ANDROID_TABLETS=m
 CONFIG_FW_ATTR_CLASS=m
 CONFIG_INTEL_IPS=m
 CONFIG_INTEL_SCU_IPC=y
@@ -8180,7 +7933,6 @@
 CONFIG_CROS_EC=m
 CONFIG_CROS_EC_I2C=m
 # CONFIG_CROS_EC_ISHTP is not set
-CONFIG_CROS_EC_SPI=m
 CONFIG_CROS_EC_LPC=m
 CONFIG_CROS_EC_PROTO=y
 CONFIG_CROS_KBD_LED_BACKLIGHT=m
@@ -8200,7 +7952,6 @@
 CONFIG_MLXREG_LC=m
 # CONFIG_NVSW_SN2201 is not set
 CONFIG_SURFACE_PLATFORMS=y
-CONFIG_SURFACE3_WMI=m
 CONFIG_SURFACE_3_POWER_OPREGION=m
 CONFIG_SURFACE_ACPI_NOTIFY=m
 CONFIG_SURFACE_AGGREGATOR_CDEV=m
@@ -8216,7 +7967,6 @@
 CONFIG_HAVE_CLK=y
 CONFIG_HAVE_CLK_PREPARE=y
 CONFIG_COMMON_CLK=y
-# CONFIG_LMK04832 is not set
 # CONFIG_COMMON_CLK_MAX9485 is not set
 # CONFIG_COMMON_CLK_SI5341 is not set
 # CONFIG_COMMON_CLK_SI5351 is not set
@@ -8376,54 +8126,39 @@
 #
 # Accelerometers
 #
-# CONFIG_ADIS16201 is not set
-# CONFIG_ADIS16209 is not set
 CONFIG_ADXL313=m
 CONFIG_ADXL313_I2C=m
-CONFIG_ADXL313_SPI=m
 # CONFIG_ADXL345_I2C is not set
-# CONFIG_ADXL345_SPI is not set
 CONFIG_ADXL355=m
 CONFIG_ADXL355_I2C=m
-CONFIG_ADXL355_SPI=m
 CONFIG_ADXL367=m
-CONFIG_ADXL367_SPI=m
 CONFIG_ADXL367_I2C=m
 CONFIG_ADXL372=m
-CONFIG_ADXL372_SPI=m
 CONFIG_ADXL372_I2C=m
 # CONFIG_BMA180 is not set
-# CONFIG_BMA220 is not set
 # CONFIG_BMA400 is not set
 CONFIG_BMC150_ACCEL=m
 CONFIG_BMC150_ACCEL_I2C=m
-CONFIG_BMC150_ACCEL_SPI=m
-# CONFIG_BMI088_ACCEL is not set
 CONFIG_DA280=m
 CONFIG_DA311=m
 # CONFIG_DMARD06 is not set
 # CONFIG_DMARD09 is not set
 CONFIG_DMARD10=m
 # CONFIG_FXLS8962AF_I2C is not set
-# CONFIG_FXLS8962AF_SPI is not set
 CONFIG_HID_SENSOR_ACCEL_3D=m
 CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
 CONFIG_IIO_ST_ACCEL_3AXIS=m
 CONFIG_IIO_ST_ACCEL_I2C_3AXIS=m
-CONFIG_IIO_ST_ACCEL_SPI_3AXIS=m
 # CONFIG_KXSD9 is not set
 CONFIG_KXCJK1013=m
 # CONFIG_MC3230 is not set
 # CONFIG_MMA7455_I2C is not set
-# CONFIG_MMA7455_SPI is not set
 CONFIG_MMA7660=m
 CONFIG_MMA8452=m
 # CONFIG_MMA9551 is not set
 # CONFIG_MMA9553 is not set
 CONFIG_MXC4005=m
 CONFIG_MXC6255=m
-# CONFIG_SCA3000 is not set
-CONFIG_SCA3300=m
 # CONFIG_STK8312 is not set
 # CONFIG_STK8BA50 is not set
 # end of Accelerometers
@@ -8431,71 +8166,32 @@
 #
 # Analog to digital converters
 #
-CONFIG_AD_SIGMA_DELTA=m
 # CONFIG_AD7091R5 is not set
-CONFIG_AD7124=m
-# CONFIG_AD7192 is not set
-# CONFIG_AD7266 is not set
-# CONFIG_AD7280 is not set
 # CONFIG_AD7291 is not set
-CONFIG_AD7292=m
-# CONFIG_AD7298 is not set
-# CONFIG_AD7476 is not set
 # CONFIG_AD7606_IFACE_PARALLEL is not set
-# CONFIG_AD7606_IFACE_SPI is not set
-CONFIG_AD7766=m
-# CONFIG_AD7768_1 is not set
-# CONFIG_AD7780 is not set
-# CONFIG_AD7791 is not set
-# CONFIG_AD7793 is not set
-# CONFIG_AD7887 is not set
-# CONFIG_AD7923 is not set
-CONFIG_AD7949=m
 # CONFIG_AD799X is not set
 # CONFIG_AXP20X_ADC is not set
 CONFIG_AXP288_ADC=m
 # CONFIG_CC10001_ADC is not set
 CONFIG_DLN2_ADC=m
-# CONFIG_HI8435 is not set
 # CONFIG_HX711 is not set
 # CONFIG_INA2XX_ADC is not set
 CONFIG_INTEL_MRFLD_ADC=m
 # CONFIG_LTC2471 is not set
 # CONFIG_LTC2485 is not set
-# CONFIG_LTC2496 is not set
 # CONFIG_LTC2497 is not set
-# CONFIG_MAX1027 is not set
-# CONFIG_MAX11100 is not set
-# CONFIG_MAX1118 is not set
-CONFIG_MAX1241=m
 CONFIG_MAX1363=m
 # CONFIG_MAX9611 is not set
-# CONFIG_MCP320X is not set
 # CONFIG_MCP3422 is not set
-CONFIG_MCP3911=m
 # CONFIG_NAU7802 is not set
 # CONFIG_TI_ADC081C is not set
-# CONFIG_TI_ADC0832 is not set
-# CONFIG_TI_ADC084S021 is not set
-# CONFIG_TI_ADC12138 is not set
-# CONFIG_TI_ADC108S102 is not set
-CONFIG_TI_ADC128S052=m
-# CONFIG_TI_ADC161S626 is not set
 CONFIG_TI_ADS1015=m
-# CONFIG_TI_ADS7950 is not set
-CONFIG_TI_ADS8344=m
-# CONFIG_TI_ADS8688 is not set
-# CONFIG_TI_ADS124S08 is not set
-CONFIG_TI_ADS131E08=m
-# CONFIG_TI_TLC4541 is not set
-CONFIG_TI_TSC2046=m
 # CONFIG_XILINX_XADC is not set
 # end of Analog to digital converters
 
 #
 # Analog to digital and digital to analog converters
 #
-CONFIG_AD74413R=m
 # end of Analog to digital and digital to analog converters
 
 #
@@ -8507,8 +8203,6 @@
 #
 # Amplifiers
 #
-# CONFIG_AD8366 is not set
-# CONFIG_ADA4250 is not set
 CONFIG_HMC425=m
 # end of Amplifiers
 
@@ -8525,7 +8219,6 @@
 # CONFIG_ATLAS_EZO_SENSOR is not set
 CONFIG_BME680=m
 CONFIG_BME680_I2C=m
-CONFIG_BME680_SPI=m
 # CONFIG_CCS811 is not set
 # CONFIG_IAQCORE is not set
 CONFIG_PMS7003=m
@@ -8560,53 +8253,26 @@
 #
 # SSP Sensor Common
 #
-# CONFIG_IIO_SSP_SENSORHUB is not set
 # end of SSP Sensor Common
 
 CONFIG_IIO_ST_SENSORS_I2C=m
-CONFIG_IIO_ST_SENSORS_SPI=m
 CONFIG_IIO_ST_SENSORS_CORE=m
 
 #
 # Digital to analog converters
 #
-CONFIG_AD3552R=m
 # CONFIG_AD5064 is not set
-# CONFIG_AD5360 is not set
 # CONFIG_AD5380 is not set
-# CONFIG_AD5421 is not set
 # CONFIG_AD5446 is not set
-# CONFIG_AD5449 is not set
-# CONFIG_AD5592R is not set
 # CONFIG_AD5593R is not set
-# CONFIG_AD5504 is not set
-# CONFIG_AD5624R_SPI is not set
-CONFIG_LTC2688=m
-# CONFIG_AD5686_SPI is not set
 # CONFIG_AD5696_I2C is not set
-# CONFIG_AD5755 is not set
-# CONFIG_AD5758 is not set
-# CONFIG_AD5761 is not set
-# CONFIG_AD5764 is not set
-CONFIG_AD5766=m
-CONFIG_AD5770R=m
-# CONFIG_AD5791 is not set
-CONFIG_AD7293=m
-# CONFIG_AD7303 is not set
-# CONFIG_AD8801 is not set
 CONFIG_DPOT_DAC=m
 # CONFIG_DS4424 is not set
-CONFIG_LTC1660=m
-# CONFIG_LTC2632 is not set
 # CONFIG_M62332 is not set
 # CONFIG_MAX517 is not set
 # CONFIG_MAX5821 is not set
 # CONFIG_MCP4725 is not set
-# CONFIG_MCP4922 is not set
-# CONFIG_TI_DAC082S085 is not set
 # CONFIG_TI_DAC5571 is not set
-CONFIG_TI_DAC7311=m
-# CONFIG_TI_DAC7612 is not set
 # CONFIG_VF610_DAC is not set
 # end of Digital to analog converters
 
@@ -8619,7 +8285,6 @@
 #
 # Filters
 #
-# CONFIG_ADMV8818 is not set
 # end of Filters
 
 #
@@ -8629,42 +8294,26 @@
 #
 # Clock Generator/Distribution
 #
-# CONFIG_AD9523 is not set
 # end of Clock Generator/Distribution
 
 #
 # Phase-Locked Loop (PLL) frequency synthesizers
 #
-# CONFIG_ADF4350 is not set
-# CONFIG_ADF4371 is not set
-# CONFIG_ADMV1013 is not set
-# CONFIG_ADMV1014 is not set
-# CONFIG_ADMV4420 is not set
-# CONFIG_ADRF6780 is not set
 # end of Phase-Locked Loop (PLL) frequency synthesizers
 # end of Frequency Synthesizers DDS/PLL
 
 #
 # Digital gyroscope sensors
 #
-# CONFIG_ADIS16080 is not set
-# CONFIG_ADIS16130 is not set
-# CONFIG_ADIS16136 is not set
-# CONFIG_ADIS16260 is not set
-CONFIG_ADXRS290=m
-# CONFIG_ADXRS450 is not set
 CONFIG_BMG160=m
 CONFIG_BMG160_I2C=m
-CONFIG_BMG160_SPI=m
 CONFIG_FXAS21002C=m
 CONFIG_FXAS21002C_I2C=m
-CONFIG_FXAS21002C_SPI=m
 CONFIG_HID_SENSOR_GYRO_3D=m
 CONFIG_MPU3050=m
 CONFIG_MPU3050_I2C=m
 CONFIG_IIO_ST_GYRO_3AXIS=m
 CONFIG_IIO_ST_GYRO_I2C_3AXIS=m
-CONFIG_IIO_ST_GYRO_SPI_3AXIS=m
 # CONFIG_ITG3200 is not set
 # end of Digital gyroscope sensors
 
@@ -8675,7 +8324,6 @@
 #
 # Heart Rate Monitors
 #
-# CONFIG_AFE4403 is not set
 # CONFIG_AFE4404 is not set
 CONFIG_MAX30100=m
 # CONFIG_MAX30102 is not set
@@ -8692,7 +8340,6 @@
 CONFIG_HID_SENSOR_HUMIDITY=m
 CONFIG_HTS221=m
 CONFIG_HTS221_I2C=m
-CONFIG_HTS221_SPI=m
 # CONFIG_HTU21 is not set
 # CONFIG_SI7005 is not set
 # CONFIG_SI7020 is not set
@@ -8701,31 +8348,19 @@
 #
 # Inertial measurement units
 #
-# CONFIG_ADIS16400 is not set
-# CONFIG_ADIS16460 is not set
-CONFIG_ADIS16475=m
-# CONFIG_ADIS16480 is not set
 # CONFIG_BMI160_I2C is not set
-# CONFIG_BMI160_SPI is not set
 CONFIG_FXOS8700=m
 CONFIG_FXOS8700_I2C=m
-CONFIG_FXOS8700_SPI=m
 # CONFIG_KMX61 is not set
 CONFIG_INV_ICM42600=m
 CONFIG_INV_ICM42600_I2C=m
-CONFIG_INV_ICM42600_SPI=m
 CONFIG_INV_MPU6050_IIO=m
 CONFIG_INV_MPU6050_I2C=m
-# CONFIG_INV_MPU6050_SPI is not set
 CONFIG_IIO_ST_LSM6DSX=m
 CONFIG_IIO_ST_LSM6DSX_I2C=m
-CONFIG_IIO_ST_LSM6DSX_SPI=m
 # CONFIG_IIO_ST_LSM9DS0 is not set
 # end of Inertial measurement units
 
-CONFIG_IIO_ADIS_LIB=m
-CONFIG_IIO_ADIS_LIB_BUFFER=y
-
 #
 # Light sensors
 #
@@ -8766,7 +8401,6 @@
 CONFIG_STK3310=m
 CONFIG_ST_UVIS25=m
 CONFIG_ST_UVIS25_I2C=m
-CONFIG_ST_UVIS25_SPI=m
 # CONFIG_TCS3414 is not set
 # CONFIG_TCS3472 is not set
 # CONFIG_SENSORS_TSL2563 is not set
@@ -8791,18 +8425,14 @@
 # CONFIG_AK09911 is not set
 CONFIG_BMC150_MAGN=m
 CONFIG_BMC150_MAGN_I2C=m
-# CONFIG_BMC150_MAGN_SPI is not set
 # CONFIG_MAG3110 is not set
 CONFIG_HID_SENSOR_MAGNETOMETER_3D=m
 # CONFIG_MMC35240 is not set
 CONFIG_IIO_ST_MAGN_3AXIS=m
 CONFIG_IIO_ST_MAGN_I2C_3AXIS=m
-CONFIG_IIO_ST_MAGN_SPI_3AXIS=m
 # CONFIG_SENSORS_HMC5843_I2C is not set
-# CONFIG_SENSORS_HMC5843_SPI is not set
 CONFIG_SENSORS_RM3100=m
 CONFIG_SENSORS_RM3100_I2C=m
-CONFIG_SENSORS_RM3100_SPI=m
 # CONFIG_YAMAHA_YAS530 is not set
 # end of Magnetometer sensors
 
@@ -8841,12 +8471,8 @@
 CONFIG_AD5272=m
 # CONFIG_DS1803 is not set
 # CONFIG_MAX5432 is not set
-# CONFIG_MAX5481 is not set
-# CONFIG_MAX5487 is not set
 CONFIG_MCP4018=m
-# CONFIG_MCP4131 is not set
 # CONFIG_MCP4531 is not set
-CONFIG_MCP41010=m
 # CONFIG_TPL0102 is not set
 # end of Digital potentiometers
 
@@ -8862,7 +8488,6 @@
 CONFIG_ABP060MG=m
 CONFIG_BMP280=m
 CONFIG_BMP280_I2C=m
-CONFIG_BMP280_SPI=m
 CONFIG_IIO_CROS_EC_BARO=m
 # CONFIG_DLHL60D is not set
 # CONFIG_DPS310 is not set
@@ -8871,7 +8496,6 @@
 CONFIG_ICP10100=m
 CONFIG_MPL115=m
 CONFIG_MPL115_I2C=m
-# CONFIG_MPL115_SPI is not set
 # CONFIG_MPL3115 is not set
 # CONFIG_MS5611 is not set
 # CONFIG_MS5637 is not set
@@ -8884,7 +8508,6 @@
 #
 # Lightning sensors
 #
-# CONFIG_AS3935 is not set
 # end of Lightning sensors
 
 #
@@ -8910,15 +8533,11 @@
 #
 # Resolver to digital converters
 #
-# CONFIG_AD2S90 is not set
-# CONFIG_AD2S1200 is not set
 # end of Resolver to digital converters
 
 #
 # Temperature sensors
 #
-CONFIG_LTC2983=m
-CONFIG_MAXIM_THERMOCOUPLE=m
 CONFIG_HID_SENSOR_TEMP=m
 CONFIG_MLX90614=m
 CONFIG_MLX90632=m
@@ -8927,8 +8546,6 @@
 CONFIG_TMP117=m
 # CONFIG_TSYS01 is not set
 # CONFIG_TSYS02D is not set
-CONFIG_MAX31856=m
-CONFIG_MAX31865=m
 # end of Temperature sensors
 
 CONFIG_NTB=m
@@ -9053,10 +8670,7 @@
 
 CONFIG_FPGA=m
 CONFIG_ALTERA_PR_IP_CORE=m
-CONFIG_FPGA_MGR_ALTERA_PS_SPI=m
 CONFIG_FPGA_MGR_ALTERA_CVP=m
-CONFIG_FPGA_MGR_XILINX_SPI=m
-CONFIG_FPGA_MGR_MACHXO2_SPI=m
 CONFIG_FPGA_BRIDGE=m
 # CONFIG_ALTERA_FREEZE_BRIDGE is not set
 CONFIG_XILINX_PR_DECOUPLER=m
@@ -9077,7 +8691,6 @@
 # Multiplexer drivers
 #
 CONFIG_MUX_ADG792A=m
-# CONFIG_MUX_ADGS1408 is not set
 CONFIG_MUX_GPIO=m
 # end of Multiplexer drivers
 
